### PR TITLE
feat(ui): improved control panel

### PR DESCRIPTION
**Pull request by robertsLando**
Based on #524

Peek 17-03-2021 16-07

### CODE REVIEW

**Comment by ahochsteger** [comment]
@robertsLando may you provide a quick summary about the major differences between #524 and this PR?

**Comment by robertsLando** [comment]
@ahochsteger 

I have took the controller status, info box (the box with the homeid and applications version) and the advanced dialog from #524. What I did the is to completely refactor how the advanced dialog works like passing actions as an arguments and handle the new action event. This makes it more easier to use in both node and controller action expecially paired with args. 

What I have not implemented by #524 is the main idea about the compact control panel table that require some more work expecially in the dialog to control node stuff (everything that ATM we have on expanded node in the table)

File: src/components/nodes-table/NodeDetails.vue [hunk 1]
```diff
@@ -120,31 +81,161 @@
...
+                confirm:
+                  'This action will remove this node from all associations with others'
+              }
+            }
+          ],
+          icon: 'link_off',
+          desc:
+            'Clear all node associations / Remove node from all associations'
+        }
+      ]
```
Comment (robertsLando) [inline src/components/nodes-table/NodeDetails.vue:238]: @AlCalzone

File: src/components/ControlPanel.vue [hunk 2]
```diff
@@ -115,61 +79,72 @@ export default {
...
+              action: 'removeFailedNode',
+              args: {
+                broadcast: true,
+                confirm: 'This action will remove all failed nodes'
+              }
+            }
+          ],
+          icon: 'dangerous',
+          desc:
+            'Manage nodes that are dead and/or marked as failed with the controller'
```
Comment (robertsLando) [inline src/components/ControlPanel.vue:147]: @AlCalzone

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: src/components/nodes-table/NodeDetails.vue [hunk 3]
```diff
@@ -120,31 +81,161 @@
...
+        },
+        {
+          text: 'Associations',
+          options: [
+            {
+              name: 'Clear',
+              action: 'removeAllAssociations',
+              args: {
+                confirm:
+                  "This action will remove all associations of this node. Does clearing a node's associations include the lifeline? If so, the node won't report until that is set up again"
```
[the remark this answers is not in the document]
Comment (robertsLando) [reply src/components/nodes-table/NodeDetails.vue]: LOL I misundestood it so, yeah it includes that, I have fixed the confirm message :laughing:

**Pull request merged**